lesson06/assignment/FAQ.html

```diff
@@ -0,0 +1,149 @@
+<!doctype html>
```

Comment:
"DOCTYPE" should be capitalized. https://developer.mozilla.org/en-US/docs/Glossary/Doctype